spec/ruby/optional/capi/ext/string_spec.c

```diff
@@ -181,6 +181,10 @@ VALUE string_spec_rb_str_length(VALUE self, VALUE str) {
 VALUE string_spec_rb_str_new(VALUE self, VALUE str, VALUE len) {
   return rb_str_new(RSTRING_PTR(str), FIX2INT(len));
 }
+
+VALUE string_spec_rb_str_new_offset(VALUE self, VALUE str, VALUE len, VALUE offset) {
```

Comment:
Minor thing but I think most API usually take (receiver, offset, length) in that order (like System.arraycopy, String#[offset, length], etc).

Reply:
Fixed.